README.md

```diff
@@ -140,16 +173,44 @@ would look like for such an application.
 
 Your description:
 ```
-Replace this with your answer
+convention
+REST is a for how information should be communicated between a browser and a server.
+Seven RESTful routes: INDEX, SHOW, NEW, CREATE, EDIT, UPDATE, DELETE.
 ```
 Your routes:
-```
-The ancestors have provided an example of one route; you do the other six!
+```ruby
+# The ancestors have provided an example of one route; you do the other six!
+
+get '/warriors/:id' do #show
+  @warrior = Warrior.find_by(id: params[:id])
+  erb :"warriors/:id"
+end
 
-GET '/warriors/:id'
-  * This is the show route, which finds a warrior by ID, and displays information about that warrior.
+get '/warriors' do #index
+  @warriors = Warrior.all
+  erb :"warriors/index"
+
+get '/warriors/new' do #new
+  erb :"warriors/new"
+end
+
+post 'warriors/new' do #create
+  Warrior.create![params[:warrior]]
+  redirect :"warriors/index"
+end
+
+destroy 'warriors/:id' do #destroy
+  Warrior.find_by(id: params[:id]).destroy!
+end
+
+put 'warriors/:id' #edit
```

Comment:
Think what really want is a `get` request to fetch the page where the form lives, then use our update action to handle submission. i.e.: 
```ruby
# edit route
get 'warriors/:id` do 
  @warrior = Warrior.find(params[:id])
  erb :"warriors/edit"
end